branchpro/models.py

```diff
@@ -97,12 +102,71 @@ def __init__(self, initial_r, serial_interval):
 
         # Invert order of serial intervals for ease in _normalised_daily_mean
         self._serial_interval = np.asarray(serial_interval)[::-1]
-        self._initial_r = initial_r
+        self._r_profile = np.array([initial_r])
         self._normalizing_const = np.sum(self._serial_interval)
 
+    def set_r_profile(self, new_rs, start_times, last_time=None):
+        """
+        Creates a new R_t profile for the model.
+
+        Parameters
+        ----------
+        new_rs: sequence of new time-dependent values of the reproduction
```

Comment:
For this docstring, putting the description after the : is causing problems with sphinx.
Easiest fix should be to start the description in the next line (with indent) and remove colon. I believe sphinx expects a type after the colon, but its not necessary.